src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -30,8 +30,9 @@ public async Task<ActionResult> UploadAutoReview([FromForm] IFormFile file, stri
                     var review = await _reviewManager.CreateMasterReviewAsync(User, file.FileName, label, openReadStream, false);
                     if(review != null)
                     {
+                        var reviewUrl = $"{this.Request.Scheme}://{this.Request.Host}/Assemblies/Review/{review.ReviewId}";
```

Comment:
There is a way to do this without hardcoding the URL: https://stackoverflow.com/questions/699782/creating-a-url-in-the-controller-net-mvc

Reply:
I will explore this option. If I am not able to make it work quickly, I would like to go ahead with current change and add an action item to replace it with above suggestion once I am back from vacation.

Reply:
@pakrym : I would like to merge this PR for now and revisit and make change as per your suggestion above. I will create an issue so it will still be tracked.

Reply:
Here is the tracking issue: https://github.com/Azure/azure-sdk-tools/issues/1332